src/kaleidoscope/key_events.cpp

```diff
@@ -137,8 +137,3 @@ void handleKeyswitchEvent(Key mappedKey, KeyAddr key_addr, uint8_t keyState) {
     return;
   handleKeyswitchEventDefault(mappedKey, key_addr, keyState);
 }
-
```

Comment:
This does not make sense here. Had to move it to the header to make the `inline` actually work.